src/transformers/training_args.py

```diff
@@ -450,6 +450,9 @@ class TrainingArguments:
         full_determinism (`bool`, *optional*, defaults to `False`)
             If `True`, [`enable_full_determinism`] is called instead of [`set_seed`] to ensure reproducible results in
             distributed training
+        torchdynamo (`str`, *optional*):
+            If `True`, TorchDynamo is called with AOT Autograd and nvfuser compiler to compile the appropriate portions
```

Comment:
this doesn't match the actual usage. Definitely not `True`/`False` but the actual choices : eager, nvfuser